### ISSUE TITLE
Fedora 35: Errors in sensors.conf display message with incorrect link to github issues

### ISSUE THREAD

**Issue by MisterGuinness** [MisterGuinness/gnome-shell-extension-sensors#63]
I thought I could write a custom sensors conf file to relabel the CPU and GPU temps on my wife's new mini PC, but alas I introduced typos all over the place.

This is what the sensors command displays:

![Screenshot from 2022-04-21 08-59-06](https://user-images.githubusercontent.com/7279446/164354225-a0db64ba-3368-46de-abf8-991380c92f4d.png)

And the popup menu now shows this:
![Screenshot from 2022-04-21 08-49-38](https://user-images.githubusercontent.com/7279446/164354019-3bc9f486-96f9-4dab-95db-3c552947c679.png)

Clicking on that text opens the browser in the original repository, so there must be an old link somewhere that I missed

**Comment by MisterGuinness** [comment]
The odd thing is that all the sensors output is there, just missing the renames & ignores etc specified in the config file containing the error. Why does the error text on the first line cause the extension to think there is no output?

**Comment by MisterGuinness** [comment]
With an invalid config, sensors on the command line still returns zero, does output the error to stderr, but the usual data also appears on stdout (minus the changes specified in the invalid config file, of course). Somehow the presence of stderr causes the stdout to not be available.

I don't see much value in puting more time into tracking that down further, although it might be useful to display the text from stderr instead of assuming sensors-detect needs to be run, or that an issue needs to be raised.

**Comment by MisterGuinness** [comment]
Of course I had to have another look at the stderr causing stdout to disappear issue, I just created a new method to read stderr based on the existing one for stdout, but just logging the text instead of invoking the callback. Immediately the stderr was logged and the sensor data appeared as usual (without the relabelling specified in the invalid config). So it seems confirmed that if there is stderr you must read it, otherwise your punishment is no stdout either.

That's all well and good, but now I have data from two async child processes, but only want one return point where the callback is invoked.
